dashboard/Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM python:3.9-slim-bullseye
 
 WORKDIR /src/dashboard
 
-RUN apt-get update -y && apt-get install -y postgresql-client
+RUN apt-get update -y && apt-get install -y postgresql-client \
```

Comment:
It's recommended to clear the apt cache at the end of update: `&& rm -rf /var/lib/apt/lists/*` , here and in other images also.

See: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get

Reply:
I think you're right. Once I skipped that on purpose due to 
> Official Debian and Ubuntu images automatically run apt-get clean, so explicit invocation is not required.

which is included in /etc/apt/apt.conf.d/docker-clean in python:3.9-slim-bullseye image.

but now I think `apt-get clean` is something different `rm -rf /var/lib/apt/lists/*` so I probably need both.

Reply:
Yeah, judging by this code: https://github.com/moby/moby/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105 , the apt cache clean is not cleaning /apt/lists/.